test/test_fourier.jl

```diff
@@ -0,0 +1,407 @@
+            times,
+            times2,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+        )
+        out_ct = avg_cs_from_events(
+            bin_times,
+            bin_times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes1=counts,
+            fluxes2=counts2,
+        )
+        if use_common_mean
+            compare_tables(out_ev, out_ct, rtol=0.01)
+        else
+            compare_tables(out_ev, out_ct, rtol=0.1)
+        end
+    end
+
+    @testset "test_avg_cs_cts_and_err_and_events_are_equal" for use_common_mean in [true,false], norm in ["frac", "abs", "none", "leahy"]
+        out_ev = avg_cs_from_events(
+            times,
+            times2,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+        )
+        out_ct = avg_cs_from_events(
+            bin_times,
+            bin_times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes1=counts,
+            fluxes2=counts2,
+            errors1=errs,
+            errors2=errs2,
+        )
+        discard = [m for m in propertynames(out_ev) if m == Symbol("variance")]
+
+        if use_common_mean
+            compare_tables(out_ev, out_ct, rtol=0.01, discard=discard)
+        else
+            compare_tables(out_ev, out_ct, rtol=0.1, discard=discard)
+        end
+    end
+end
+
+@testset "test_norm" begin
+    mean = var = 100000
+    N = 1000000
+    dt = 0.2
+    meanrate = mean / dt
+    lc = rand(Poisson(mean),N)
+    pds = abs.(fft(lc)).^2
+    freq = fftfreq(N, dt)
+    good = 2:floor(Int,N/2)
+
+    pdsabs = normalize_abs(pds, dt, size(lc,1))
+    pdsfrac = normalize_frac(pds, dt, size(lc,1), mean)
+    pois_abs = poisson_level(meanrate=meanrate, norm="abs")
+    pois_frac = poisson_level(meanrate=meanrate, norm="frac")
+
+    @test Statistics.mean(pdsabs[good])≈pois_abs rtol=0.02
+    @test Statistics.mean(pdsfrac[good])≈pois_frac rtol=0.02
+
+    mean = var = 100000.
+    N = 800000
+    dt = 0.2
+    df = 1 / (N * dt)
+    freq = fftfreq(N, dt)
+    good = freq .> 0
+    meanrate = mean / dt
+    lc = rand(Poisson(mean),N)
+    nph = sum(lc)
+    pds = (abs.(fft(lc)) .^ 2)[good]
+    lc_bksub = lc .- mean
+    pds_bksub = (abs.(fft(lc_bksub)) .^ 2)[good]
+    lc_renorm = lc / mean
+    pds_renorm = (abs.(fft(lc_renorm)) .^ 2)[good]
+    lc_renorm_bksub = lc_renorm .- 1
+    pds_renorm_bksub = (abs.(fft(lc_renorm_bksub)) .^ 2)[good]
+
+    @testset "test_leahy_bksub_var_vs_standard" begin
+        """Test that the Leahy norm. does not change with background-subtracted lcs"""
```

Comment:
This string is doing nothing.  What is is supposed to do?  A comment?

Reply:
Yes it was comment :)